styleguide/derek/global-components/filter-bar/filter-bar.js

```diff
@@ -1,13 +1,63 @@
+/* eslint no-use-before-define: 0 */
+
+const Zoolander = Zoolander || {};
+
+Zoolander.rsFilterBar = {
+  calcAtBottom: (dimensions) => {
+    const defaults = {
+      filterBarOffsetTop: 0,
+      filterBarOuterHeight: 0,
+      filterFormOuterHeight: 0,
+      mainNavHeight: 0,
+      windowTop: 0,
+    };
+    const values = Object.assign(defaults, dimensions);
+    const filterFormHeight = parseInt(values.filterFormOuterHeight, 10);
```

Comment:
@xinbin do tests still work with the `parseInt` calls removed? I suspect they are not necessary.